models/review.go

```diff
@@ -255,3 +255,22 @@ func UpdateReview(r *Review) error {
 	}
 	return nil
 }
+
+// PullReviewersWithType represents the type used to display a review overview
+type PullReviewersWithType struct {
+	User              `xorm:"extends"`
+	Type              ReviewType
+	ReviewUpdatedUnix util.TimeStamp `xorm:"review_updated_unix"`
+}
+
+// GetReviewersByPullID gets all reviewers for a pull request with the statuses
+func GetReviewersByPullID(pullID int64) (issueReviewers []*PullReviewersWithType, err error) {
+	err = x.Select("`user`.*, review.type, max(review.updated_unix) as review_updated_unix").
```

Comment:
You can not select something that is not in group by

Reply:
Why not? It works.

Reply:
Just tested with postgres and first I got `pq: column user.id does not exist` fixed it with escaping user table in this code:
```
Join("INNER", "`user`", "review.reviewer_id = `user`.id").
```
As I suspected now I get:
```
[E] GetReviewersByPullID: pq: column "review.type" must appear in the GROUP BY clause or be used in an aggregate function
```
I think same will be with other user table columns being selected

Reply:
I think we should get the last review type since one could review multiple times?

Reply:
@lunny Thats what I do now.
@lafriks I've fixed it for postgresql, but apparantly I had to also group by `type` and then create a map afterwards to have only the newest review per user.